src/videojs.ads.js

```diff
@@ -462,9 +443,11 @@ var
             enter: function() {
               // capture current player state snapshot (playing, currentTime, src)
               this.snapshot = getPlayerSnapshot(player);
-
               // remove the poster so it doesn't flash between videos
               removeNativePoster(player);
+              // We no longer need to supress play events once an ad is playing
```

Comment:
Not following.  This is where the code from ad-timeout-playback was moved to (which was invoked after putting it into content-playback state).  I am making the assumption (perhaps incorrectly) that we would never want to supress play events while content is playing.  Is this not true?